dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataImport.java

```diff
@@ -785,16 +785,18 @@ private void buildRelationObject(Context c, Item item, String value, boolean lef
      *                                          for the right entity
      * @param relationshipType                  The RelationshipType object that we want to check whether it's
      *                                          valid to be added or not
-     * @return                                  A boolean indicating whether the relationship is left or right, will
+     * @return A boolean indicating whether the relationship is left or right, will
      *                                          be false in this case
      * @throws SQLException                     If something goes wrong
      */
-    private boolean handleRightLabelEqualityRelationshipTypeElement(Context c, Entity entity, Entity relationEntity,
-                                                                    boolean left,
-                                                                    List<RelationshipType> acceptableRelationshipTypes,
-                                                                    List<RelationshipType>
-                                                                        rightRelationshipTypesForEntity,
-                                                                    RelationshipType relationshipType)
+    private boolean handleRightwardTypeEqualityRelationshipTypeElement(Context c, Entity entity,
+                                                                       Entity relationEntity,
+                                                                       boolean left,
```

Comment:
Well, it *is* used *if* no relationship is added, because in that case it is passed through as the value of the method.  If I've worked out the truth table correctly, the method returns `left && !(relationship is right)`.  What that *means* is more than I can say at present.

Reply:
That's correct.
If no relationship matched, it is unmodified
If a relationship matched, the parameter is set to the direction of that relationship
This has been documented more extensively now